src/components/FilterBar/ExportResultsList.react.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types";
+import t from "../../locales/i18n";
+
+export class ExportResultsList extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  onClick(option) {
+    this.context.filterBarActor.exportResults(option);
+  }
+
+  render() {
+    return (
+      <div className="btn-group">
+        <button
+          aria-expanded="false"
+          aria-haspopup="true"
+          className="btn btn-default"
+          data-toggle="dropdown"
+          type="button"
+        >
+          <i className="icon icon-download" />
+          {t('filterbar.buttons.export')}
+          <i className="icon icon-chevron-down" />
+        </button>
+        <ul className="dropdown-menu" role="menu">
+          {this.props.filterBarActor.filterBarStore.exportAllOptions === 'true' &&
```

Comment:
you can move this line logic to the FilerBar.react.js

Reply:
@hanpeic https://github.com/rdytech/react-filterbar/pull/138/commits/d6444960a04ceb4eed95d604253b2a12a297fa8d